packages/wonder-stuff-core/src/__tests__/values.test.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import {values} from "../values.js";
+
+describe("#values", () => {
+    it("should call Object.values with the given object", () => {
+        // Arrange
+        const valuesSpy = jest.spyOn(Object, "values");
+        const obj = {
+            a: 1,
+            b: "2",
+            c: [3, 4],
+        };
+
+        // Act
+        values(obj);
+
+        // Assert
+        expect(valuesSpy).toHaveBeenCalledWith(obj);
+    });
+
+    it("should return the result of Object.values", () => {
+        // Arrange
+        const obj = {
+            a: 1,
+            b: "2",
+            c: [3, 4],
+        };
+        jest.spyOn(Object, "values").mockReturnValue([1, 2, 3]);
+
+        // Act
+        const result = values(obj);
+
+        // Assert
+        expect(result).toEqual([1, 2, 3]);
+    });
```

Comment:
What's the purpose of mocking `Object.values` here? IMO a non-mocked test would:

- illustrate usage and expected results better, by showing real inputs and outputs.
- avoid coupling the tests to the implementation (perhaps letting us refactor to a different implementation with test coverage, if by some chance we find that Object.values doesn't do what we want).

Perhaps, though, we want these tests to enforce the invariant that `values` is a pass-through wrapper around `Object.values`, since that's how we want to define `values`. If that's the case, I think the tests could use a comment (or maybe different test titles?) explaining the intent.

Reply:
It seems appropriate to just test that it returns what `Object.values` returns rather to retest what that should be, especially since that's our aim with these wrappers.

I think the test title explains that. It says that it should return what `Object.values` returns and it does just that. I don't think there's anything to adjust here.